### PR TITLE
Government interventions validations

### PR DESCRIPTION
**Pull request by ilcsik**

### CODE REVIEW

**Comment by ilcsik** [comment]
You can start reviewing this as I only have an issue regarding testing so I'm leaving WIP label on.

File: EquiTrack/EquiTrack/validation_mixins.py [hunk 1]
```diff
@@ -187,12 +187,15 @@ def __init__(self, new, user=None, old=None, instance_class=None):
             new = new_instance
             old = old_instance
 
+        self.stateless = stateless
```
Comment (robertavram) [inline EquiTrack/EquiTrack/validation_mixins.py]: good idea.

File: EquiTrack/partners/models.py [hunk 2]
```diff
@@ -1749,7 +1749,7 @@ def save(self, **kwargs):
 
 def activity_default():
     return {
-        "0": "",
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: activity Default should be an empty dict

Comment (ilcsik) [inline EquiTrack/partners/models.py]: Done in https://github.com/unicef/etools/pull/408/commits/a30b79be34d0e94e92f62813465b7569bc92bbd5

**Comment by ntrncic** [comment]
@ilcsik looking good! Is this finished?

**Comment by ilcsik** [comment]
@ntrncic I haven't been able to resolve the test data creation issue yet (cannot use Sections in tests since it's in the public schema). Asked some help from simon but he also had no idea :)

Due to this, Section validation in GovIntResults disabled currently. Apart from this, it's done.

**Comment by ntrncic** [comment]
@ilcsik please add a check to see if the partner.type is Government. If it is not do not allow create or update.

**Comment by ilcsik** [comment]
Done in https://github.com/unicef/etools/pull/408/commits/3271d44aee3eccb7eb404883e5b1caa5bb8adf18

**Pull request merged**